src/key.js

```diff
@@ -51,14 +54,18 @@ class Key {
    * @param {string} [encoding='utf8']
    * @returns {string}
    */
-  toString (encoding) {
-    return this._buf.toString(encoding || 'utf8')
+  toString (encoding = 'utf8') {
+    if (encoding === 'utf8' || encoding === 'utf-8') {
+      return utf8Decoder.decode(this._buf)
+    }
+
+    return new TextDecoder(encoding).decode(this._buf)
   }
 
   /**
-   * Return the buffer representation of the key
+   * Return the Uint8Array representation of the key
    *
-   * @returns {Buffer}
+   * @returns {Uint8Array}
    */
   toBuffer () {
     return this._buf
```

Comment:
📝 I think this name is misleading now, maybe it should be renamed to `toBytes` or to `toUint8Array` ?

Reply:
Web APIs (e.g. [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob/arrayBuffer), [Response](https://developer.mozilla.org/en-US/docs/Web/API/Body/arrayBuffer)) on the other hand seem to have `arrayBuffer()` method that returns `ArrayBuffer` which might be another option worth considering.

Reply:
A good suggestion, I've changed the method name to `Key.uint8Array()`.

I want to like the idea of `.arrayBuffer()` for symmetry/sympathy with other browser APIs but I'm struggling to see the utility of returning the underlying `ArrayBuffer` as the view is what we're interested in and there's a whole bunch of weirdness around what to do when the byteLengths are different.

I guess we can add it if we find that we need it.